5.1.no_vinaigrette.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+import random
+
+
+def no_vinaigrette():       # from 5.1
+    date1 = input("Enter the first date in format YYYY-MM-DD:")
+    date2 = input("Enter the second date in format YYYY-MM-DD:")
+    date1 = datetime.strptime(date1, '%Y-%m-%d').date()
+    date2 = datetime.strptime(date2, '%Y-%m-%d').date()

```

Comment:
Rename 'date' to 'start_date' and 'end_date'

Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing